index.html

```diff
@@ -12,8 +12,164 @@
+                    sites with a mobile first approach and ensure that our code quality is clean, organized
+                    and
+                    performant. Already have a design but need it developed? We’re here to help!</p></article>
+            <article class="service-card">
+                <img src="assets/icon_ux.svg" alt="user experience">
+                <h3>User Experience</h3>
+                <p>In order to build successful products you have to understand the user. We create user
+                    personas,
+                    wireframes and prototypes to test with users. This allows us to gather feedback which
+                    helps to
+                    shape the product and can quickly eliminate potential risks – saving you time and
+                    money.</p>
+            </article>
+            <article class="service-card">
+                <img src="assets/icon_graphic-design.svg" alt="graphic design">
+                <h3>Logo Design + Branding</h3>
+                <p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily
+                    recognizable
+                    logos and brand identity. The best brands stand out from the crowd by having a clear and
+                    unique
+                    point of view. We will work with you to understand your vision and help create a unique
+                    brand
+                    identity and style guide for your company.</p></article>
+            <article class="service-card">
+                <img src="assets/icon_seo.svg" alt="search engine optimization">
+                <h3>SEO</h3>
+                <p>Do you want your site to rank higher in search results? Google is constantly changing their
+                    algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will
+                    make
+                    sure your website it optimized, providing you the best opportunities for organic search
+                    exposure.</p></article>
+            <article class="service-card">
+                <img src="assets/icon_copywriting.svg" alt="copy writing">
+                <h3>Copy Writing</h3>
+                <p>Brand messaging needs to be clear, direct and easy to understand to be effective. A unique
+                    tone
+                    of
+                    voice will help you stand out from the crowd and build brand loyalty. We work with you to
+                    establish your brand bible and ensure your messaging is focused and targeted.</p>
+            </article>
+        </div>
+    </div>
+</section>
+
+<section id="contact" class="contact outer-container">
+    <div class="inner-container-sm">
+        <div class="form-title-container">
+            <h2>Let's work together</h2>
+            <p>Have a question? Need a quote? Let us know!</p>
+        </div>
+        <form action="">
+            <label for="name">Name</label>
```

Comment:
Het is semantisch beter om de label te rappen om je input heen, zodat eventuele eigenschappen die op de input van toepassing zijn, ook op de label effect hebben. 

Een goed voorbeeld hiervan is de checkbox. Wanneer je deze binnen een label plaatst, kun je ook op de naam van het label klikken om de checkbox aan en uit te zetten.